### PR TITLE
Remove obsolete comment

### PR DESCRIPTION
**Pull request by suragch**
The method `_stopCursorTimer` no longer exists. The comment line that mentions it is now misleading.

- This PR removes the obsolete comment line.

Alternatively, replace the comment with the following:

```
// Cannot use _stopCursorBlink because it would reset _cursorActive.
```

Because it appears that `_stopCursorBlink` also resets `_cursorActive`.

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [ ] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [ ] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: packages/flutter/lib/src/widgets/editable_text.dart [hunk 1]
```diff
@@ -2605,7 +2605,6 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
       if (_tickersEnabled && _cursorActive) {
         _startCursorBlink();
       } else if (!_tickersEnabled && _cursorTimer != null) {
-        // Cannot use _stopCursorTimer because it would reset _cursorActive.
```
Comment (tgucio) [inline packages/flutter/lib/src/widgets/editable_text.dart:2608]: Shouldn't this be changed to  -> `_stopCursorBlink?` IMHO the comment is useful as looking at the likes above it'd be tempting to do the opposite of `_startCursorBlink()`

```
  void _stopCursorBlink({ bool resetCharTicks = true }) {
    _cursorActive = false;
    _cursorBlinkOpacityController.value = 0.0;
    _cursorTimer?.cancel();
    _cursorTimer = null;
    if (resetCharTicks) {
      _obscureShowCharTicksPending = 0;
    }
  }
```

Comment (justinmc) [reply packages/flutter/lib/src/widgets/editable_text.dart:2608]: I think you're right and the comment should remain, but should say _stopCursorBlink.

**Comment by goderbauer** [comment]
(Triage): @suragch Do you have plans to follow up on the feedback given above?

**Comment by suragch** [comment]
@goderbauer followup on feedback finished

**Review by goderbauer — APPROVED** [review]
LGTM

**Review by Piinks — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**